<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>840</width>
         <height>584</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="6" column="3">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="3">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="nameTitleLabel">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string>Your name</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="nameLabel">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string>fullName</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="balanceTitleLabel">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string>Your current balance</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="balanceLabel">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string>balance</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="6" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QSpinBox" name="withdrawSpinBox">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="maximum">
             <number>20000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="withdrawButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>100</height>
             </size>
            </property>
            <property name="text">
             <string>Withdraw</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="6" column="4">
         <widget class="QPushButton" name="depositButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string>Deposit</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QLineEdit" name="pinInput"/>
        </item>
        <item row="4" column="4">
         <widget class="QPushButton" name="pinSubmitButton">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Submit</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <layout class="QVBoxLayout" name="reportHandler">
          <item>
           <widget class="QLabel" name="reportConfirmationLabel">
            <property name="text">
             <string>Do you want to generate a report?</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDialogButtonBox" name="reportConfirmationButtons">
            <property name="standardButtons">
             <set>QDialogButtonBox::No|QDialogButtonBox::Yes</set>
            </property>
            <property name="centerButtons">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
